src/main/java/no/unit/nva/doi/FetchDoiMetadata.java

```diff
@@ -29,7 +29,7 @@
     public static final String MISSING_ACCEPT_HEADER = "Missing Accept header";
     public static final Pattern DOI_URL_PATTERN =
         Pattern.compile("^https?://(dx\\.)?doi\\.org/(10(?:\\.[0-9]+)+)/(.+)$",
-                        Pattern.CASE_INSENSITIVE);
+            Pattern.CASE_INSENSITIVE);
```

Comment:
Pattern matching DOIs https://www.crossref.org/blog/dois-and-matching-regular-expressions/

Reply:
Not your commit, I know, but I prefer to see regular expressions outside the code. Either on paper in the bottom of a locked filing cabinet stuck in a disused lavatory with a sign on the door saying 'Beware of the Leopard', or failing that at least in a well-named constant.

Reply:
Create an issue for this?

Reply:
https://unit.atlassian.net/browse/NP-691